plugins/plugin-jenkins/codenvy-plugin-jenkins-webhooks/src/main/java/com/codenvy/plugin/jenkins/webhooks/shared/JenkinsEvent.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  [2012] - [2017] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.plugin.jenkins.webhooks.shared;
+
+import org.eclipse.che.dto.shared.DTO;
+
+/**
+ * DTO object for describing Jenkins failed build event.
+ *
+ * @author Igor Vinokur
+ */
+@DTO
+public interface JenkinsEvent {
```

Comment:
As far as I know we should add `Dto` suffix to dto interfaces. Please rename it to JenkinsEventDto if I am right.